metadata.yaml

```diff
@@ -1,15 +1,24 @@
-# Copyright 2023 Canonical Ltd.
+# Copyright 2024 Canonical Ltd.
```

Comment:
**question**: Are we required to keep those up to date?

Reply:
It's usually the year the file was created. For code that ends up being the _Ship of Theseus_ I believe it should be a range? So `# Copyright 2023-2024 Canonical Ltd.` But I don't think we follow any convention yet on our codebases.

Reply:
We could do this on another PR I suppose. Otherwise the CI will re-run only for this.